.circleci/config.yml

```diff
@@ -32,26 +32,18 @@ jobs:
             # download custom jsign to sign PE
             curl -fsSL https://get.rookout.com/jsign-rookout.jar --output jsign.jar
             # install yarn
-            npm install -g yarn
+            npm install -g yarn@1.17.3
             # install electron main window dependencies
             yarn
             # build code (mostly compile typescript)
             yarn run build
-            # go to react windows folder
-            cd src/webapp
             # install dependencies
-            yarn
             # CircleCI sets CI=true which makes react return an error when there are warnings in the code
-            export CI=false
-            # build react app
-            yarn run build
-            # copy react bundle to dist directory
-            mv build/* ../../dist/
-            cd ../..
+            CI=false npm run --prefix src/webapp build
```

Comment:
yarn ?

Reply:
Doesn't matter `yarn run` and `npm run` are equivalent

Reply:
Switched to `yarn` for consistency :)